lib/file.go

```diff
@@ -44,11 +45,30 @@ func FileCopy(dest string, source string) error {
 	}
 	defer d.Close()
 
-	err = d.Chmod(fi.Mode())
+	if mode != nil {
```

Comment:
Hm, this isn't a bug in this patch per se, but i do notice that in the context up above if the copied file is a symlink, you do not chown it.  That could provide wrong results.  It would be best to LChown() it to linkFI's uid/gid.

Reply:
2022-12-13T19:42:34.4519138Z not ok 16 importing broken symlink is ok
2022-12-13T19:42:34.4519524Z # (from function `stacker' in file test/helpers.bash, line 86,
2022-12-13T19:42:34.4519909Z #  in test file test/broken-link.bats, line 24)
2022-12-13T19:42:34.4520230Z #   `stacker build' failed